model/marketplace_plugin.go

```diff
@@ -74,11 +74,13 @@ func (plugin *BaseMarketplacePlugin) DecodeSignature() (io.ReadSeeker, error) {
 
 // MarketplacePluginFilter describes the parameters to request a list of plugins.
 type MarketplacePluginFilter struct {
-	Page          int
-	PerPage       int
-	Filter        string
-	ServerVersion string
-	LocalOnly     bool
+	Page              int
+	PerPage           int
+	Filter            string
+	ServerVersion     string
+	EEReady           bool
```

Comment:
Can we use `BuildEnterpriseReady` to match `model.BuildEnterpriseReady`?

Reply:
That makes sense, thanks for the suggestion :+1: